docker-compose.yml

```diff
@@ -55,15 +55,11 @@ services:
       context: docker/images/fakesentry
     image: local/eliot_fakesentry
     ports:
-      - "8090:8090"
+      - "${EXPOSE_SENTRY_PORT:-8090}:8090"
```

Comment:
these values can be set in `.env`, which allows me to set it to something different in tecken/eliot/socorro for vscode